interni/norme-di-progetto/components/processi-primari/sviluppo.tex

```diff
@@ -542,15 +542,74 @@ \subsubsection{Attività}%
 L'uso della ricorsione andrebbe motivato con appositi commenti.
 % subp:ricorsione (end)
 
+\subsubsection{Metriche}%
+\label{subs:sviluppo/metriche}
+
+\paragraph{MPS-ROS: Requisiti obbligatori soddisfatti}% in questo caso NON ci va il "\@". chktex 13
+\label{par:MPS-ROS_requisiti_obbligatori_soddisfatti}
+
+La metrica requisiti obbligatori soddisfatti indica la percentuale di requisiti obbligatori soddisfatti sul numero totale di requisiti obbligatori. La formula è la seguente:
+\[
+  \frac{requisiti\ obbligatori\ soddisfatti}{requisiti\ obbligatori\ totali}\cdot 100
+\]
+
+\paragraph{MPS-NFV: Nomi di file e variabili non normati}% in questo caso NON ci va il "\@". chktex 13
+\label{par:MPS-NFV_nomi_file_variabili_non_normati}
+
+I nomi dei file o delle variabili devono seguire delle specifiche regole per essere più chiare possibile.
+
+\textbf{Metrica:} numero di nomi di variabili o di titoli di metodi o classi che non rispettano le norme stabilite.
+
+\paragraph{MPS-CNN: Commenti non normati}% in questo caso NON ci va il "\@". chktex 13
+\label{par:MPS-CNN_commenti_non_normati}
+
+I commenti devono essere scritti seguendo le norme indicate per essere comprensibili.
+
+\textbf{Metrica:} numero di commenti che non rispettano le norme indicate.
+
+\paragraph{MPS-CCE: Complessità ciclomatica eccessiva}% in questo caso NON ci va il "\@". chktex 13
+\label{par:MPS-CCE_complessita_ciclomatica_eccessiva}
+una sezione di codice è il numero di cammini linearmente indipendenti attraverso il codice sorgente
```

Comment:
Qualcosa non va in questa frase...  Penso debba essere "La complessità ciclomatica rappresenta il numero di cammini eccetera"